From 05cac74a57cd480543f16a3d68ce88f739ad80de Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Rafa=C5=82=20Mi=C5=82ecki?= <rafal@milecki.pl>
Date: Wed, 29 Jul 2020 10:17:18 +0200
Subject: [PATCH 1/4] cli: document usage of "subscribe" command
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Support for "subscribe" command was added years ago in the commit
453b87f6315f ("cli: add support for subscribing to objects"). Document
its usage.

Cc: Felix Fietkau <nbd@nbd.name>
Signed-off-by: Rafał Miłecki <rafal@milecki.pl>
(cherry picked from commit 59835ae530f5dae69d00dc92d201f56bcff792be)
---
 cli.c | 1 +
 1 file changed, 1 insertion(+)

--- a/cli.c
+++ b/cli.c
@@ -549,6 +549,7 @@ static int usage(const char *prog)
 		"Commands:\n"
 		" - list [<path>]			List objects\n"
 		" - call <path> <method> [<message>]	Call an object method\n"
+		" - subscribe <path> [<path>...]	Subscribe to object(s) notifications\n"
 		" - listen [<path>...]			Listen for events\n"
 		" - send <type> [<message>]		Send an event\n"
 		" - wait_for <object> [<object>...]	Wait for multiple objects to appear on ubus\n"
